src/System.Diagnostics.Tools/tests/System.Diagnostics.Tools.Tests.builds

```diff
@@ -2,11 +2,15 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Diagnostics.Tools.Tests.csproj"/>
+  <Project Include="System.Diagnostics.Tools.Tests.csproj"/>
     <Project Include="System.Diagnostics.Tools.Tests.csproj">
       <TestTFMs>netcore50;net46</TestTFMs>
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Diagnostics.Tools.Tests.csproj">
```

Comment:
@joperezr  you are going to need to pick-up this change with your PR https://github.com/dotnet/corefx/pull/12241.